### PR TITLE
Refactor/cleanup

### PR DESCRIPTION
**Pull request by lilioid**
you might need to adapt cronjobs because the `extract.sh` file is no longer present

### CODE REVIEW

File: generate_extracts.py [hunk 1]
```diff
@@ -1,55 +1,98 @@
...
+                        help='Target size which the split files should have')
+    return parser.parse_args()
+
+
+def download_planet_dump(url: str, working_dir: Path):
+    print(f'Downloading {url}')
+    subprocess.run(['wget', '-N', '-nv', '--show-progress', url], check=True, cwd=str(working_dir))
+
+    file_name = url.rsplit('/', 1)[-1]
+    subprocess.run(['ln', '-sf', (working_dir / file_name).absolute(), (working_dir / 'source.osm.pbf').absolute()])
```
Comment (Akasch) [inline generate_extracts.py]: I think you want to name the file 0_0_0.pbf or you have to change the code below accessing the file 0_0_0.pbf

**Review by Akasch — CHANGES_REQUESTED** [review]
Maybe merge it with #1 for the parallel processing as here it will wait on all tiles of a zoom level to have finished, while in #1 the calculation of the children begins immediately if processes are free, speeding up the whole process because otherwise most of the time all Threads are waiting for the one having central Europe in it.

File: generate_extracts.py (hunk 1 above)
Comment (lilioid) [reply generate_extracts.py]: i have done so

**Comment by lilioid** [comment]
ping @Akasch

**Review by Akasch — APPROVED** [review]
If it is tested to be working it looks good

**Pull request merged**